docs/laravel.md

```diff
@@ -0,0 +1,55 @@
+# Laravel
+
+## Docker
+
+Serving a [Laravel](https://laravel.com) web application with FrankenPHP is as easy as mounting the project in the `/app` directory of the official Docker image.
+
+Run this command from the main directory of your Laravel app:
+
+```console
+docker run -p 443:443 -v $PWD:/app dunglas/frankenphp
+```
+
+And enjoy!
+
+## Local Installation
+
+Alternatively, you can run your Laravel projects with FrankenPHP from your local machine:
+
+1. [Download the binary corresponding to your system](https://github.com/dunglas/frankenphp/releases)
+2. Add the following configuration to a file named `Caddyfile` in the root directory of your Laravel project:
+```Caddyfile
+{
+	frankenphp
+}
+
+# The domain name of your server
+localhost
+
+route {
```

Comment:
Instead of `route`, you can use `order php before php_fastcgi` in global options. Removes a level of nesting.